install-terraform.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# installs 'tfenv', a Terraform version manager: https://github.com/tfutils/tfenv
+# this allows us to migrate projects between versions of Terraform.
+
+set -e
+
+if [ ! -d .tfenv ]; then
+    wget https://github.com/tfutils/tfenv/archive/refs/tags/v3.0.0.tar.gz \
```

Comment:
stick this on our CDN? I feel a bit wrong with CI pulling this from github all the time.